python/py-country/Portfile

```diff
@@ -17,16 +17,21 @@ long_description    {*}${description}.
 
 homepage            https://github.com/flyingcircusio/pycountry
 
-checksums           rmd160  aae4bc0dd11033b2e99c499ec29c54a992928a4d \
-                    sha256  b9a6d9cdbf53f81ccdf73f6f5de01b0d8493cab2213a230af3e34458de85ea32 \
-                    size    10060795
+checksums           rmd160  4232fb90e97eba3d7a6cdf1f49e2d149184640ab \
+                    sha256  b2163a246c585894d808f18783e19137cb70a0c18fb36748dc01fc6f109c1646 \
+                    size    10141551
 
 python.versions     37 38 39 310
 
+patch.pre_args      -p1
+patchfiles          do-not-file-without-package.diff
```

Comment:
did you mean `do-not-fail-without-package.diff` as filename?

Reply:
@reneeotten yes, I did. Thanks to catch it.